src/fheroes2/heroes/heroes_meeting.cpp

```diff
@@ -325,9 +333,20 @@ void Heroes::MeetingDialog( Heroes & otherHero )
     std::string message( _( "%{name1} meets %{name2}" ) );
     StringReplace( message, "%{name1}", GetName() );
     StringReplace( message, "%{name2}", otherHero.GetName() );
-    const fheroes2::Text text( message, fheroes2::FontType::normalWhite() );
+    fheroes2::Text text( message, fheroes2::FontType::normalWhite() );
```

Comment:
We can pass `message` without copying it:
```suggestion
    fheroes2::Text text( std::move( message ), fheroes2::FontType::normalWhite() );
```
The same is for lines 342 and 347.